### PR TITLE
added propaganda about the tower ToCMC from JToH zone 7

### PR DESCRIPTION
**Pull request by heheboi6**

### CODE REVIEW

File: ToCMC.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ToCMC so good the humour is so good the gameplay is
```
Comment (carol8) [inline ToCMC.txt:1]: Ce e cu linia asta ba aici?

**Review by carol8 — CHANGES_REQUESTED** [review]
Nigga rezolva aia

File: ToCMC.txt (hunk 1 above)
Comment (heheboi6) [reply ToCMC.txt:1]: propaganda

**Review by carol8 — APPROVED** [review]
All good

**Pull request merged**